doc/documentation.rst

```diff
@@ -106,5 +106,12 @@ Documentation of scikit-learn |version|
                     <blockquote>Roadmap of the project.
                     </blockquote>
                 </div>
+            </div>
+            <div class="row-fluid">
+                <div class="span4 box">
+                    <h2><a href="governance.html">Governance</a></h2>
```

Comment:
Hmm... I had thought it sufficient to link to this from the About Us page, but now I can't see where we link to About Us, except on index.html

Reply:
I don't think anywhere except for "about us". The whole navigation is a bit of a mess tbh, I don't have strong feelings either way

Reply:
I think we somehow came to the conclusion to not put the link here. But I personally prefer to have it here and more visible.

Reply:
We should probably link to About Us here instead of governance. And we should possibly move the Governance section of about.rst up to before the author list???